kickstart/scripts/samba-deploy.sh

```diff
@@ -11,6 +11,18 @@ deploy_samba_ubuntu() {
   chown nobody:nogroup /opt/data/fieldpapers
   chmod 777 /opt/data/fieldpapers
 
+  mkdir -p /opt/data/backups/omk
+  chown nobody:nogroup /opt/data/backups/omk
+  chmod 777 /opt/data/backups/omk
```

Comment:
Do we want to have nobody own it and give full rights to the whole system? Is there a reason behind this?